charts/kube-state-metrics/templates/serviceaccount.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
   annotations:
 {{ toYaml .Values.serviceAccount.annotations | indent 4 }}
 {{- end }}
+{{- if .Values.serviceAccount.imagePullSecrets }}
```

Comment:
This should also include global image pull secrets:
https://github.com/prometheus-community/helm-charts/blob/11127a45423d6cf468e476e9ee5a800b7a6c29af/charts/kube-state-metrics/templates/deployment.yaml#L250